app/src/app/components/view-profile/view-profile.component.scss

```diff
@@ -0,0 +1,172 @@
+adf-view-profile {
+  letter-spacing: .5px;
+
+  .profile_container{
+    margin-top: 1rem;
+    overflow: scroll;
+    height:100%;
+    width: 100%;
+  }
+
+  .profile_row{
+    width: 100%;
+  }
+
+  .profile_title{
+    display: flex;
+    margin-left: 2rem;
+  }
+
+  .profile{
+    cursor: pointer;
+  }
+
+  .profile_general_row{
+    margin: 2rem 0 0 2rem;
+    width: 70%;
+    border: 1px solid rgba(33,33,33,0.12);
+    border-radius: 1rem;
+  }
+
+  .profile_icon{
+    margin-right: 1rem;
+    margin-top: 1rem;
+    cursor: pointer;
+  }
+
+  .profile_general_icon{
+    cursor: pointer;
+  }
+
+  .profile_text{
+    letter-spacing: .5px;
+  }
+
+  .profile_general{
+    display: flex;
+    padding-left: 1rem;
+  }
+
+  .profile_general_bottom_radius{
+  border-bottom-left-radius:0;
+  border-bottom-right-radius: 0;
+  }
+
+  .profile_general_div{
+    display: flex;
+    width: 60%;
+    padding-top: 1rem;
+    cursor: pointer;
+  }
+
+  .general_title{
+    margin-left: 0.6rem;
+    margin-top: 4px;
+    letter-spacing: .5px;
+  }
+
+  .general_edit_btn{
+    width:60%;
+    text-align: end;
+  }
+
+  .mat_divider{
+    border-top-width: 2px;
+    border-top-color: rgba(33,33,33,0.12);
```

Comment:
colors should be coming from the CSS variables

Reply:
Done.